optimum/gptq/quantizer.py

```diff
@@ -137,6 +141,11 @@ def __init__(
             raise ValueError("group_size must be greater than 0 or equal to -1")
         if not (0 < self.damp_percent < 1):
             raise ValueError("damp_percent must between 0 and 1.")
+        if not self.disable_exllamav2 and not self.disable_exllama:
+            logger.warning(
+                "You have activated exllama and exllamav2 backend. Setting `disable_exllama=True` and keeping `disable_exllamav2=False`"
+            )
+            self.disable_exllama = True
```

Comment:
I think you should raise an error here, but it is personal taste.

Reply:
Yeah, good point. The user will have to be more careful.